specs/deposits.md

```diff
@@ -257,7 +257,7 @@ The deposit contract handles two special cases:
 
 If the caller is a contract, the address will be transformed by adding
 `0x1111000000000000000000000000000000001111` to it. This is the reverse of the
-transformation described in the [deposits spec](./withdrawals.md#address-aliasing). This prevents attacks in which a
+transformation described in the [withdrawals spec](./withdrawals.md#address-aliasing). This prevents attacks in which a
```

Comment:
Note that we do not do this transformation for withdrawals anymore. Another spec change to watch out for.